### DISCUSSION TITLE
Vagrant problems

### DISCUSSION THREAD

**Discussion by profesyonal** [Tatoeba/tatoeba2#3106] [General]
Hi,

I had a Vagrant server up, and it was starting to give me a problem where it said "Call to undefined function __nx()" in the title and then gave a non-stylized version of sentence and other pages.

I did `vagrant reload` to see if I'd still have the same problem, and I did.

Then I did `vagrant destroy` and `vagrant up`. I logged into the Contributor account, tried to add English as a language, and it said "You cannot save this language." Does anyone know why this could be?

I don't understand the problem well enough to open an issue about it yet. Particularly, I'm not sure whether anyone would be able to reproduce this problem or not. Also, it may be two problems.

**Comment by profesyonal** [comment]
I did another `vagrant destroy` and `vagrant up`, and the problem with adding languages was fixed. I'll update about the __nx() problem if it reoccurs.

**Reply by profesyonal** [reply]
I ran into the "You cannot save this language" error again, but it only happens sometimes. I have no idea when.

**Comment by profesyonal** [comment]
It's reoccurring. I'm going to open up an issue on it.

**Comment by jiru** [comment]
Thank you for helping us improve Tatoeba :+1:
